packages/cli/src/zosfiles/create/pds/Pds.definition.ts

```diff
@@ -47,6 +47,7 @@ export const PdsDefinition: ICommandDefinition = {
         {...ZosFilesCreateOptions.recfm, defaultValue: CreateDefaults.DATA_SET.PARTITIONED.recfm},
         {...ZosFilesCreateOptions.blksize, defaultValue: CreateDefaults.DATA_SET.PARTITIONED.blksize},
         {...ZosFilesCreateOptions.lrecl, defaultValue: CreateDefaults.DATA_SET.PARTITIONED.lrecl},
+        {...ZosFilesCreateOptions.lrecl, defaultValue: CreateDefaults.DATA_SET.PARTITIONED.alcunit},
```

Comment:
Should this be `ZosFilesCreateOptions.alcunit`?

Reply:
I think the spread operator and `CreateDefaults` should be kept like below, sorry for the confusion:
```
{...ZosFilesCreateOptions.alcunit, defaultValue: CreateDefaults.DATA_SET.PARTITIONED.alcunit}
```

The reason for keeping the `CreateDefaults` is that it defines a default value for `alcunit` here: https://github.com/zowe/zowe-cli/blob/master/packages/zosfiles/src/methods/create/Create.defaults.ts#L25